### DISCUSSION TITLE
How to Update Hacktribe

### DISCUSSION THREAD

**Discussion by tripsylon** [bangcorrupt/hacktribe#281] [Q&A] [answered]
Hello please where can I find the new update file, and do I need to do something  with it, or just simply drag it to the card and do the update? Thank you, I have Hacktribe installed on the red sampler.

**Comment by bangcorrupt** [comment, marked as answer]
Use the [Hacktribe Tutorial](https://github.com/bangcorrupt/hacktribe-tutorial) to get the update file.  Put `sampler-hacked-SYSTEM.VSB` in the `KORG/hacktribe/System/` directory and rename it to `SYSTEM.VSB`. Then run the update function on the device.

(If you are running a very old version of Hacktribe, you may need to put the update file in the `'KORG/electribe sampler/System/'` directory.)

**Reply by tripsylon** [reply]
THAKS A TON!!! IT WORKED :)
ARE ALL THE UPDATES GOING TO BE THIS WAY?

**Reply by bangcorrupt** [reply]
Yes, this process will work every time the patch is updated.

**Comment by der-narr** [comment]
Hello community,

I'm looking for information regarding the original folder structure for the gray Electribe Synth (Music Production Station) firmware. I suspect that my directory structure might be incorrect, which is preventing the successful installation of the system.vsb file.

Could someone who has experience with the original firmware of the gray Electribe Synth kindly provide details about how the folders were organized? Specifically, I'm interested in the main directory and any subdirectories where the system.vsb file was located.

I'm attempting to revert to the original firmware, and having accurate information about the folder structure would greatly help me in this process.

Your assistance is much appreciated. Thank you!

**Reply by bangcorrupt** [reply]
`SYSTEM.VSB` should always be at `KORG/<DEVICE>/System/SYSTEM.VSB`, where `<DEVICE>` is `hacktribe`, `electribe` or `electribe sampler`.  There should be no other files in the `System` directory (sub-directories are OK).  If you backup your SD card then format the card on the device, it will create the correct directory structure for the firmware you have installed.  The structure is the same for each device, you can copy the `hacktribe` directory and rename it to `electribe`.  

Recent versions of Hacktribe will look for files under the `hacktribe` directory.  Very old versions of Hacktribe will look for files under the`electribe sampler` directory.  In most cases, formatting the card will create the correct directory for your firmware version.  If you're not sure, use both paths. 

To revert from Hacktribe to Electribe Synth:

1. Use the [Hacktribe Tutorial](https://github.com/bangcorrupt/hacktribe-tutorial) to get the update files.
2. Put `synth-revert-SYSTEM.VSB` at `KORG/hacktribe/System/SYSTEM.VSB` (or  `KORG/electribe sampler/System/SYSTEM.VSB`).
4. Run the firmware update function on the device.

**Reply by der-narr** [reply]
Thx ... but ;-),

I wanted to share my recent experience updating the firmware on my Korg Electribe 2. Although I encountered an "error" message during the process, the update was successful, and my Electribe 2 is now running the latest firmware.

Despite the error message, I followed the instructions carefully, and it seems that the firmware update completed without any issues. My Electribe 2 is functioning perfectly with the new firmware, and I'm excited to explore the new features and improvements.

If anyone else encounters a similar "error" message during a firmware update, don't panic right away. It's possible that the update was successful, as was the case for me. Of course, it's essential to double-check and ensure that your device is working correctly after the update.

Thank you to this community for all the helpful advice and information throughout this process.

Best regards,
